server/src/ankaios_server/server_state.rs

```diff
@@ -199,6 +204,33 @@ impl ServerState {
             .collect()
     }
 
+    fn update_delete_graph(&mut self, added_workloads: &[WorkloadSpec]) {
+        for workload_spec in added_workloads {
+            for (dependency_name, add_condition) in workload_spec.dependencies.iter() {
+                /* currently for other add conditions besides AddCondRunning
+                the workload can be deleted immediately and does not need a delete condition */
+                if add_condition == &AddCondition::AddCondRunning {
+                    let workload_name = workload_spec.name.clone();
+                    // [impl->swdd~server-state-stores-delete-condition~1]
```

Comment:
I would put the linkage as a comment to the function (and maybe also at the place it is called).